### PR TITLE
Release 2.0

### PR DESCRIPTION
**Pull request by fulldecent**
Tracking PR for everything that needs to be done to release version 2.0 to general availability.

### CODE REVIEW

**Comment by MoMannn** [comment]
TODO before merge: 

- [ ] Add all intended functionalities
- [ ] Update documentation
- [ ] Full code audit
- [ ] Deploy smart contracts and add them to documentation
- [ ] Bump version

**Comment by fulldecent** [comment]
The

```
\```sh
```

changing to

```
\```
```

is no good!

Ignore the backslash in both.

File: packages/0xcert-cert/src/core/cert.ts [hunk 1]
```diff
@@ -416,7 +416,7 @@ export class Cert {
    */
   protected sortSchema(obj) {
     return Object.keys(obj).sort().reduce((acc, key) => {
-      if (Array.isArray(obj[key])) { // sport arrays
+      if (Array.isArray(obj[key])) { // suport arrays
```
Comment (davision) [inline packages/0xcert-cert/src/core/cert.ts]: suport -> support

Comment (xpepermint) [reply packages/0xcert-cert/src/core/cert.ts]: already fixed in branch 2.0

**Comment by xpepermint** [comment]
Not sure if this PR is still relevant. Feel free to reopen it.

**Pull request closed without merging**